src/genkei/cli/etf_flows.py

```
@@ -285,6 +288,133 @@ def _format_per_ticker_human(asset: str, rows: list[dict[str, Any]], horizon_tag
 
 
+def _query_net_flow(
+    asset: str,
+    *,
+    since: Optional[date],
+    until: Optional[date],
+    limit: int,
+) -> list[dict[str, Any]]:
+    """Return signed-net-flow rows per BlackRock ETF for one underlying asset.
+
+    Net flow is derived at query time via ``(shares − LAG(shares)) × nav``.
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)